CoreLayersView/build.xml

```diff
@@ -3,6 +3,6 @@
 <!-- for some information on what you could do (e.g. targets to override). -->
 <!-- If you delete this file and reopen the project it will be recreated. -->
 <project name="Core Layers View" default="netbeans" basedir=".">
-    <description>Builds, tests, and runs the project au.gov.asd.tac.</description>
```

Comment:
A load of these updates were caused by that issue I had with the module renaming. I'm just not sure what went wrong.